test/integration/smart_answer_flows/am_i_getting_minimum_wage_test.rb

```diff
@@ -658,11 +914,11 @@ class AmIGettingMinimumWageTest < ActiveSupport::TestCase
       add_response 'no'
 
       assert_current_node :past_payment_below
-      assert_equal 6.08, current_state.calculator.minimum_hourly_rate # rate on '2011-10-01'
+      assert_equal 6.19, current_state.calculator.minimum_hourly_rate # rate on '2012-10-01'
 
-      expected_underpayment = 46.18
+      expected_underpayment = 49.98
       # (hours worked * hourly rate back then - paid by employer) / minimum hourly rate back then * minimum hourly rate today
-      # (40h * £6.08 - £200.0) / 6.08 * 6.50 = 46.18
+      # (40h * £6.08 - £200.0) / 6.19 * 6.50 = 46.18
```

Comment:
Shouldn't this be `(40h * £6.19 - £200.0) / 6.19 * 6.50 = 49.98` ?

Reply:
oops, yes, cursed comments.